pkg/transfers/inbound/returns.go

```diff
@@ -89,7 +93,7 @@ func (pc *returnProcessor) processReturnEntry(fh ach.FileHeader, bh *ach.BatchHe
 	// Do we find a Transfer related to the ach.EntryDetail?
 	transfer, err := pc.transferRepo.LookupTransferFromReturn(amount, entry.TraceNumber, effectiveEntryDate)
 	if transfer != nil {
-		pc.logger.Log("inbound", fmt.Sprintf("handling return for transferID=%s", transfer.TransferID))
+		pc.logger.Set("transferID", transfer.TransferID).Log("handling return for transfer")
```

Comment:
I would create new logger `logger := pc.logger.Set("transferID", transfer.TransferID)` and use it in line 115 (with more context).